### ISSUE TITLE
FPS limitation?

### ISSUE THREAD

**Issue by lujeff** [yomotsu/camera-controls#140]
Hello!

I'm having an issue similar to #85 where the framerate drops when rotating the camera. I am also using [react-three-fiber](https://github.com/pmndrs/react-three-fiber) and I've tried that fix you commented on the issue above but it didn't seem to work.

It becomes extremely slow on Firefox dropping down to 10-20 FPS, where on Edge/Chrome it'll be around 40 - 55 FPS. 

I also do use 240hz and 144hz monitors so I suspect that might one of the issues why it's dropping down to around 60 since I noticed that you use some sort of constant for 60 FPS.

Was wondering if there was any possible fix for this issue? Thanks!

**Comment by yomotsu** [comment]
I don't see the problem.

<img width="1680" alt="Screen Shot 2020-10-04 at 1 37 19 AM" src="https://user-images.githubusercontent.com/212837/94996875-29fb1280-05e2-11eb-8097-84e791906c26.png">

**Comment by lujeff** [comment]
Hmm, you're right. I completely forgot to check the examples before making this issue. It's probably something on my end. Sorry about that and thanks!

**Comment by yomotsu** [comment]
That's okay!
If you find any problem with the lib, let's continue the investigation.

**Comment by lujeff** [comment]
I seem to have gotten it working, the actual issue I found that was causing the slow framerate was a conflict from the postprocessing lib from [react-postprocessing](https://github.com/pmndrs/react-postprocessing). I think the EffectComposer was conflicting with the camera rendering in the code I used from #85. I removed the gl.render from the camera object and just let the EffectComposer handle the rendering. 

It seems to be working now and was completely unrelated to what I thought it was. All in all, extremely excited to get your library running and can't wait to see what I can do with it! :)
